http3 will not build for wasm32-unknown-unknown after recent dep changes

0.12.19 builds fine, 0.12.20 doesn't

https://github.com/seanmonstar/reqwest/pull/2716

error[E0432]: unresolved import `slab`
   --> /home/tux/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/reqwest-0.12.20/src/lib.rs:261:5
    |
261 | use slab as _;
    |     ^^^^^^^^^ no external crate `slab`
For more information about this error, try `rustc --explain E0432`.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -84,7 +84,7 @@ system-proxy = ["hyper-util/client-proxy-system"]
 macos-system-configuration = ["system-proxy"]
 
 # Experimental HTTP/3 client.
-http3 = ["rustls-tls-manual-roots", "dep:h3", "dep:h3-quinn", "dep:quinn", "dep:slab", "tokio/macros"]
+http3 = ["rustls-tls-manual-roots", "dep:h3", "dep:h3-quinn", "dep:quinn", "tokio/macros"]
 
 
 # Internal (PRIVATE!) features used to aid testing.
@@ -163,7 +163,6 @@ once_cell = { version = "1.18", optional = true }
 h3 = { version = "0.0.8", optional = true }
 h3-quinn = { version = "0.0.10", optional = true }
 quinn = { version = "0.11.1", default-features = false, features = ["rustls", "runtime-tokio"], optional = true }
-slab = { version = "0.4.9", optional = true } # just to get minimal versions working with quinn
 futures-channel = { version = "0.3", optional = true }
 
 [target.'cfg(not(target_arch = "wasm32"))'.dev-dependencies]

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -256,9 +256,6 @@ compile_error!(
 );
 
 // Ignore `unused_crate_dependencies` warnings.
-// Used to pin the version.
-#[cfg(feature = "http3")]
-use slab as _;
 // Used in many features that they're not worth making it optional.
 use futures_core as _;
 use sync_wrapper as _;

